crates/sui-adapter/src/adapter.rs

```diff
@@ -57,23 +60,25 @@ use tracing::instrument;
 
 use crate::execution_mode::{self, ExecutionMode};
 
+static VERIFIER_CONFIG: Lazy<VerifierConfig> = Lazy::new(|| VerifierConfig {
+    max_loop_depth: Some(MAX_LOOP_DEPTH),
+    max_generic_instantiation_length: Some(MAX_GENERIC_INSTANTIATION_LENGTH),
+    max_function_parameters: Some(MAX_FUNCTION_PARAMETERS),
+    max_basic_blocks: Some(MAX_BASIC_BLOCKS),
+    max_value_stack_size: MAX_VALUE_STACK_SIZE,
+    max_type_nodes: Some(MAX_TYPE_NODES),
+    max_push_size: Some(MAX_PUSH_SIZE),
+    max_dependency_depth: Some(MAX_DEPENDENCY_DEPTH),
+    max_fields_in_struct: Some(MAX_FIELDS_IN_STRUCT),
+    max_function_definitions: Some(MAX_FUNCTION_DEFINITIONS),
+    max_struct_definitions: Some(MAX_STRUCT_DEFINITIONS),
+});
```

Comment:
Can't this just be a `const`?

Reply:
I would also make it public and clear it is the source of truth for anything Sui related. 
Because those configs are going to be vital for the functioning of the system.
We can cleanup another time but let's keep it in mind. It's true that Move specific tests are in the Move repo, so using this config may not be vital.
Not sure, thoughts?